src/enhancement/equalization.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+def histogram_equalization(image, number_bins=256):
+    # Use numpy to compute normalized image histogram
```

Comment:
Could you add documentation to this function? If possible, using [Google docstring format](https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e), for compatibility with the starter code. What do you think?

Also, could you add one usage example of this function to test it's correctness? Thanks!